### PR TITLE
added enzyme tests for Sizes page

### LINKED ISSUE #604: (SP:5)create enzyme tests for 'Sizes' page

### PR DESCRIPTION
**Pull request by ostgit12**
Adds working enzyme tests for component 'Sizes' of folder pages.
Closes #604

### CODE REVIEW

**Comment by IlyaBelik** [comment]
image
Please link an issue

File: src/pages/sizes/tests/sizes.test.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    },
+    Table: {
+      pagination: {
+        currentPage: 0,
+        rowsPerPage: 10
+      },
+      itemsCount: 1
+    }
+  };
+}
```
Comment (IlyaBelik) [inline src/pages/sizes/tests/sizes.test.js:67]: Please make a sizes.variables.js and get these values from it

File: src/pages/sizes/tests/sizes.test.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import { config } from '../../../configs';
+
+let mockLoading = true;
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const { CREATE_SIZE_TITLE } = config.buttonTitles;
+
+function mockReturner() {
+  return {
```
Comment (IlyaBelik) [inline src/pages/sizes/tests/sizes.test.js:22]: please use arrow function instead of default function

File: src/pages/sizes/tests/sizes.test.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { mount, shallow } from 'enzyme';
+import { useSelector, useDispatch } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Typography, Button } from '@material-ui/core';
+import * as reactRedux from 'react-redux';
+import Sizes from '../sizes';
```
Comment (Haytham1337) [inline src/pages/sizes/tests/sizes.test.js:8]: devide your imports by from general imports like " react, redux, etc"  by 1 line interval

**Comment by Kiyooooko** [comment]
we don't use this board 😑 
Screenshot 2021-03-22 at 23 06 20

File: src/pages/sizes/tests/sizes.test.js (hunk 1 above)
Comment (ostgit12) [reply src/pages/sizes/tests/sizes.test.js]: made additional file sizes.variables.js with function mockReturner() inside it.

File: src/pages/sizes/tests/sizes.test.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
```
Comment (dmytro07) [inline src/pages/sizes/tests/sizes.test.js:2]: Remove useDispatch because you don't use it in your code

**Comment by dmytro07** [comment]
Please, fix code smells

**Review by dmytro07 — APPROVED** [review]

**Review by Haytham1337 — APPROVED** [review]

**Review by IlyaBelik — APPROVED** [review]

**Pull request merged**